tests/unit/Trackable.js

```diff
@@ -201,9 +202,10 @@ define([
 
 				var observations = [];
 				// temporarily hide this so we don't load everything
+				var fetchSync = bigFiltered.fetchSync;
 				bigFiltered.fetchSync = null;
 				var bigObserved = bigFiltered.track();
-				delete bigFiltered.fetchSync;
+				bigFiltered.fetchSync = fetchSync;
```

Comment:
Can you explain to me what this change is doing?  It seems like it wouldn't have the same effect...  (Overall, I reckon the existing code is way too clever for its own good to begin with, and I'm not sure how deep that goes.)